bundles/org.openhab.binding.daikin/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -472,62 +492,98 @@
 		<label>Fan</label>
 		<description>Current fan speed setting of the AC unit</description>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
+		<tags>
+			<tag>Control</tag>
+			<tag>Level</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="airbase-acunit-zone1">
 		<item-type>Switch</item-type>
 		<label>Zone 1</label>
 		<description>Zone 1 for the AC unit</description>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Wind</tag>
```

Comment:
Do we have this `Wind` tag documented somewhere? I can imagine a weather device measuring wind, but controlling wind? Isn't it more like "air" being controlled rather than "wind"? I'm not saying anything is wrong, just curious. The Danish word for wind ("vind") is strictly about the wind outside, so controlling it would be god-like. 😉

Reply:
LOL! Inside the house, you can play god and control the wind with openHAB! ;)

These channels controls the air outlet that goes into each room. You can turn it on or shut it off. Some fancier system can let you control the amount of opening too.

I must admit it isn't ideal, but given what's available in the existing list of properties... that's the closest I can think of. The other alternative is `Opening` maybe?

If I were to invent a new Property, it might be one of these:
- `AirOutlet`
- `Air`
- `Airflow`

Reply:
BTW `Wind` is from here https://github.com/openhab/openhab-core/blob/fd53c4cb1d772c8086e7f4651f19db20bfcadfe6/bundles/org.openhab.core.semantics/model/SemanticTags.csv#L47

Reply:
> If I were to invent a new Property, it might be one of these:\n\nAirOutlet\nAir\nAirflow

AirOutlet may not be sufficient. I have a Danfoss Air recuperator ventilation unit where multiple outlets are connected to the same fan, so can't be individually controlled. I see you proposed Airflow, that might be just the right balance between generic and concrete.

Foot note: for Danfoss Air I can control mode and level, so actually I don't see air control being referenced.